src/app/app.component.ts

```diff
@@ -8,13 +9,20 @@ import { Component, OnInit } from '@angular/core';
 export class AppComponent implements OnInit {
   title = 'civis';
 
-  constructor(
-  ) {
+  constructor(private router: Router) {
 
   }
 
   ngOnInit() {
     this.setDisqus();
+
+    this.router.events.subscribe((evt) => {
+      if (!(evt instanceof NavigationEnd)) {
+          return;
+      }
+      window.scrollTo(0, 0)
+    });
```

Comment:
@barath46 Shouldn't this be in NavigationEnd? or NavigationStart? 
This will trigger it for all the navigation instances

Reply:
@Manzurkds Fixed bro.